test/hotspot/jtreg/runtime/MemberName/ResolvedMethodTableHash.java

```diff
@@ -35,72 +39,89 @@
 import java.util.ArrayList;
 import java.util.List;
 
-// The test generates thousands MethodHandles to the methods of the same name
-// and the same signature. This should not take too long, unless Method hash
+import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.process.ProcessTools;
+
+// The test generates a thousand MethodHandles to the methods of the same name
+// and the same signature. The rehash warning shouldn't be returned, unless Method hash
 // function takes only the name and the signature as an input.
-public class ResolvedMethodTableHash extends ClassLoader {
+public class ResolvedMethodTableHash {
 
-    // Generate a MethodHandle for ClassName.m()
-    private MethodHandle generate(String className) throws ReflectiveOperationException {
-        byte[] buf = new byte[100];
-        int size = writeClass(buf, className);
-        Class<?> cls = defineClass(null, buf, 0, size);
-        return MethodHandles.publicLookup().findStatic(cls, "m", MethodType.methodType(void.class));
-    }
+    public static class ResolvedMethodTableHashTest extends ClassLoader {
+        // Generate a MethodHandle for ClassName.m()
+        private MethodHandle generate(String className) throws ReflectiveOperationException {
```

Comment:
The indentation is wrong in line 53, 60, 70, 71, 96, 102, might be other places also.

Reply:
My indent script was a bit too agressive.